hi-ml-histopathology/src/histopathology/scripts/generate_crossval_report.py

```diff
@@ -133,7 +249,7 @@ def render_roc_and_pr_curves(report: HTMLReport, heading: str, level: int,
         args.output_dir = Path.cwd() / "outputs"
     workspace_config = Path(args.workspace_config).absolute() if args.workspace_config else None
 
-    print(f"Output dir: {args.output_dir.absolute()}")
+    print(f"Output dir: {Path(args.output_dir).absolute()}")
```

Comment:
I think `resolve` is [preferred](https://stackoverflow.com/a/44569249/3956024) over `absolute`.

Reply:
Thanks, changed now